static/js/admin/src/PagedDisplayForm.js

```diff
@@ -0,0 +1,53 @@
+define('PagedDisplayForm', [ 'jquery', 'jquery-ui', 'underscore', 'RemoteStateChangeMonitor', 'tpl!../templates/admin/pagedDisplayForm',
+        'ModalLoadingOverlay', 'AbstractForm', 'AlertHandler'],
+    function($, ui, _, RemoteStateChangeMonitor, pagedDisplayForm, ModalLoadingOverlay, AbstractForm) {
+
+        let defaultOptions = {
+            title : 'Edit View Settings',
+            createFormTemplate : pagedDisplayForm,
+            submitMethod: 'PUT'
+        };
+
+        function PagedDisplayForm(options) {
+            this.options = $.extend({}, defaultOptions, options);
+        }
+
+        PagedDisplayForm.prototype.constructor = PagedDisplayForm;
+        PagedDisplayForm.prototype = Object.create(AbstractForm.prototype);
+
+        PagedDisplayForm.prototype.preprocessForm = function() {
+            let newViewSetting = $('#view_settings_change', this.$form).val();
+            let pids = $('#paged_display_targets', this.$form).val();
+            this.action_url = `/services/api/edit/view_settings?targets=${encodeURIComponent(pids)}&direction=${encodeURIComponent(newViewSetting)}`;
```

Comment:
related to my comment below, there should be a `behavior` parameter. We might want to leave out direction for now since none of the other tickets include implementing it, but we can add it in easily enough later.